hazelcast/src/main/java/com/hazelcast/core/ReplicatedMap.java

```diff
@@ -135,7 +135,8 @@
      *     }
      * </pre>
      * Due to the lazy nature of the returned set, changes to the map (addition,
-     * removal, update) might be reflected in the set.<br/>
+     * removal, update) might be reflected in the collection.<br/>
+     * Changes on the map is <b>NOT</b> reflected in the collection on the <b>CLIENT</b> or vice versa.
```

Comment:
I think it should be plural everywhere: Changes on the map _are_ not reflected...

Reply:
Changes on the map are not.......reflected to the collection....

Reply:
@Serdaro Fixed.